trappy/ftrace.py

```diff
@@ -200,6 +200,8 @@ def contains_unique_word(line, unique_words=cls_for_unique_word.keys()):
             pid = int(fields_match.group('pid'))
             cpu = int(fields_match.group('cpu'))
             timestamp = float(fields_match.group('timestamp'))
+            if not fields_match.group('us'):
```

Comment:
Could do with a comment here explaining that the timestamp is either [s].[us] or [ns].